app/Makefile

```diff
@@ -53,6 +53,11 @@ DEFINES += SUPPORT_SR25519
 $(info ************  SR25519 ENABLED ************)
 endif
 
+ifeq ($(SUBSTRATE_PARSER_FULL),1)
+DEFINES += SUBSTRATE_PARSER_FULL
+$(info ************ FULL PARSER ENABLED ************)
+endif
+
 ifndef COIN
```

Comment:
Let's bump the version number

Reply:
Already bumped to 6.28.1 (up patch version).  is that ok ?

Reply:
sorry, I didn't notice! That's perfect then